yarn-project/simulator/src/client/client_execution_context.ts

```diff
@@ -133,31 +138,52 @@ export class ClientExecutionContext extends ViewDataOracle {
   }
 
   /**
-   * Return the encrypted logs emitted during this execution.
+   * Return the note encrypted logs emitted during this execution.
    */
-  public getEncryptedLogs() {
-    return this.encryptedLogs;
+  public getNoteEncryptedLogs() {
```

Comment:
Think we can just return all the logs emitted from this execution. Otherwise it could be confusing to people that it does not return logs chopped in this and the nested calls. But does return those chopped in later function calls.

Reply:
Makes sense! Will implement and chop logs from the parent level using `chopNoteEncryptedLogsFromNested()` as well